lib/Monitor/Staging.pm

```diff
@@ -91,9 +91,14 @@ sub find_live {
             } else {
                 if (! $run_row->folder_name ) {
                   warn qq[Folder name in db not available, will try to update using '$run_folder'.];
+                  # check the id_run corresponds to a run with a status of run pending
```

Comment:
```suggestion
                  # check the id_run corresponds to a run with a status of "run pending" to reduce the
                  # chance of incorrect updates if an incorrect id_run is entered by the loaders e.g. missing
                  # digit so that it matches an old run which is "run canceled" such that the automatic run
                  # folder deletion process would start removing it...
```